soc/arm64/renesas_rcar/gen3/pinctrl_soc.h

```diff
@@ -1,119 +1,11 @@
 /*
- * Copyright (c) 2021 IoT.bzh
+ * Copyright (c) 2024 EPAM Systems
```

Comment:
please keep both copyright

```suggestion
 * Copyright (c) 2021 IoT.bzh
 * Copyright (c) 2024 EPAM Systems
```

Reply:
done